### DISCUSSION TITLE
Question : how to close all issue with a specific label

### DISCUSSION THREAD

**Discussion by guidtz** [python-gitlab/python-gitlab#1761] [Q&A]
## Description of the problem, including code/CLI snippet
Hello, I want to make a cron tab qith a script to remove issue with specific label after 5 days of creation. 
There is a good method to do that ?

## Specifications

  - python-gitlab version: 2.0.1
  - API version you are using (v3/v4): I don't know
  - Gitlab server version (or gitlab.com): latest

Regards

**Comment by JohnVillalovos** [comment]
I think you will need to write a program to do it.

Probably want to read the upstream docs:
https://docs.gitlab.com/ee/api/issues.html

My guess would be you need to first find the issues that meet your criteria and then delete them.

**Comment by nejch** [comment]
Agree with @JohnVillalovos here. Something like:

```python
project = gl.projects.get(your_project)
issues = project.issues.list(all=True, created_before=your_date, labels=your_label)

for issue in issues:
    issue.state_event = "close"
    issue.save()
```

You'll have to provide `your_project`, `your_label` and an ISO 8601 date for `your_date`, should be easy in a step before that.
